apps/src/code-studio/pd/facilitator_program_registration/DateConfirm.jsx

```diff
@@ -0,0 +1,178 @@
+import React from 'react';
+import {
+  ControlLabel,
+  FormGroup,
+} from 'react-bootstrap';
+
+import ProgramRegistrationComponent from './ProgramRegistrationComponent';
+
+class DeclineTrainingDate extends ProgramRegistrationComponent {
+  renderNext() {
+    let followup;
+    if (this.props.data.declineTrainingDate === 'I want to participate in the program, but I\'m no longer able to attend these dates.') {
+      const name = (this.props.course === 'CSD') ? "csdAlternateTrainingDate" : "cspAlternateTrainingDate";
+
+      followup = this.buildButtonsFromOptions({
+        name: name,
+        label: "I am instead available to attend the following Facilitator-in-Training workshop (TeacherCon Part 2):",
+        type: 'check'
+      });
+    }
+
+    return (
+      <FormGroup>
+        {followup}
+        <p>
+          Thank you for letting us know. You do not need to complete the rest of
+          this form. Please click to submit your form.
```

Comment:
This is a little weird, but it's the copy I was given, and it ends up rendering directly before the submit button: 

![image](https://cloud.githubusercontent.com/assets/244100/24933923/c5ef0566-1ecd-11e7-84a0-1b44f063a32d.png)

I think it works